plugins/rest_connector/api/tasks.py

```diff
@@ -70,12 +70,11 @@ def perform_request(self, connector_id: str, db_id: int) -> str:
     print(body)
 
     parsed_headers = parse_headers(BytesIO(headers.encode()))
+    endpoint_url = render_template_sandboxed(connector["endpoint_url"], request_variables)
```

Comment:
TODO: We need to be able to match this to an openapi endpoint which may not be easy/possible with jinja templates...